### DISCUSSION TITLE
Self hosting b800.org?

### DISCUSSION THREAD

**Discussion by berlin4apk** [ipxe/ipxe#446] [Q&A] [answered]
Hey @NiKiZe is it possible to self hosting the b800.org site?

I tryed it with different options to get it with the public flag, but it was not working as expected.
Also the memcheck just hanging for me.

**Comment by NiKiZe** [comment, marked as answer]
My goal was to have [b800.org](https://b800.org/) up on github with "forkme" and all the bells and whistles, but there is to much that is in need of a cleanup.
I do have some parts that maybe could be made available.

Could you give any more information on what you are trying to do? And from where?
If you with "memcheck"  mean https://b800.org/memtest.ipxe I did test that a couple of days ago and working fine.
One thing to remember tho is that iPXEs `${vram}` is only available in pcbios mode.


# This will only be discussed further in NiKiZe/b800#2

**Reply by NiKiZe** [reply]
@berlin4apk looking at some logs... could it be that you used http://b800.org/public=1 and not http://b800.org/?public=1 ?

I have made some changes to allow for the `/public=1` case, but I do not recommend using that.

To see the script in browser, use https://b800.org/?public=1&ipxe=1

**Reply by NiKiZe** [reply]
I'm also looking forward to hearing more about your use-case to maybe make some code more public.

**Reply by NiKiZe** [reply]
More testing
```bash
qemu-system-x86_64 -enable-kvm -M q35 -m 256 -nic user,model=virtio,tftp=.,bootfile=http://b800.org/memtest.ipxe
```
[![memtest](https://b800.org/46wPi.png)](https://b800.org/46wPi)

Works, however with `-m 128` instead memtest hangs during init.
with `-cpu host` the machine restarts during memtest init, pressing F1 does allow it to run, but I have had restarts of the machine when it is done. but last one did work

[![memtest cpu host](https://b800.org/2UzNN.png)](https://b800.org/2UzNN)

**Reply by berlin4apk** [reply]
> I'm also looking forward to hearing more about your use-case to maybe make some code more public.

My use case is to capture the result of ipxe and memtest testing on systems / hardware without LOM/iLO remote access,
as the most of my hardware have no LOM.

**Reply by stappersg** [reply]
For the record https://github.com/NiKiZe/b800/issues/2 was created thirty **minutes** ago.

![afbeelding](https://user-images.githubusercontent.com/35434094/208669394-3b331fcf-15db-40bb-b20f-3dc0436b851b.png)
